src/helpers/outputBlock.js

```diff
@@ -5,7 +5,7 @@ module.exports = function (name, opts) {
 		this.blocks = {};
 	}
 
-	const val = this.blocks[name] && this.blocks[name].length > 0 ? this.blocks[name][0] : opts.fn(this);
+	const val = this.blocks[name] && this.blocks[name].length > 0 ? this.blocks[name].join('') : opts.fn(this);
```

Comment:
`.join(' | ')` or something to indicate separate things?

Reply:
Ah, that's a good point. I suppose this is why it currently only plucks the first one.

My use case was for actual markup where it doesn't matter if there's no spacing.

Reply:
Having done a search on github, the times `defineBlock` gets used it's _mostly_ for markup (with the only exception being for `globalDataAttributes`).

Perhaps it's better then to do `.join(' ')`?